collector/hyperv.go

```diff
@@ -496,6 +512,84 @@ func NewHyperVCollector() (Collector, error) {
 			[]string{"adapter"},
 			nil,
 		),
+
+		//
+
+		VMStorageErrorCount: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, buildSubsystemName("vm_storage"), "error_count"),
+			"This counter represents the total number of errors that have occurred on this virtual device",
+			[]string{"vm_storage"},
```

Comment:
Perhaps `device` is a better label name?

Reply:
Renamed to vm_device